### DISCUSSION TITLE
[javascript deep dive] 5장. 표현식과 문

### DISCUSSION THREAD

**Discussion by rladpwl0512** [woowacourse-study/2022-javascript-bank#3] [General]
[javascript deep dive] 5장. 표현식과 문

**Comment by jswith** [comment]
값, 표현식, 문에 대해 설명해주세요

**Reply by jswith** [reply]
값 : 3, 'a', true 등
식 : a + b, 3 + 7 등
문: react에서 return 문에 if, while 등이 왜 안될까?를 생각해보자

**Comment by jswith** [comment]
표현식인 문 vs 표현식이 아닌 문

**Reply by jswith** [reply]
표현식이 아닌 문: 할당이 안된다. 값으로 평가가 안되기 때문에.
표현식인 문 : 할당이 가능하다. 값으로 평가가 되기 때문에.
